### DISCUSSION TITLE
Async build needed with memory rework branch

### DISCUSSION THREAD

**Discussion by AussieNErd** [Sporif/dxvk-async#33] [Q&A] [answered]
Hi @Sporif 

I would like to try this branch of the memory rework of dxvk here
https://github.com/doitsujin/dxvk/tree/memory-rework

I was wondering if you could kindly build it with the Async patch so I can try it. It will address an issue I have with FFIV

Thank you ☺️

**Comment by Sporif** [comment, marked as answer]
Here you go: [dxvk-async-1.9.3_r6_g08c7bab4.tar.gz](https://github.com/Sporif/dxvk-async/files/7866140/dxvk-async-1.9.3_r6_g08c7bab4.tar.gz)

**Reply by AussieNErd** [reply]
Thank you very much 🙏
